src/components/home/HomePage.scss

```diff
@@ -0,0 +1,351 @@
+      z-index: 2;
+      background-color: black;
+
+      .hamburgerIcon {
+        display: block;
+        width: 60%;
+        height: 3px;
+        position: absolute;
+        left: 50%;
+        top: 50%;
+        transform: translate(-50%, -50%);
+        background-color: white;
+        z-index: 2;
+
+        &::before,
+        &::after {
+          content: "";
+          height: 3px;
+          position: absolute;
+          background-color: white;
+          z-index: 2;
+        }
+
+        &::before {
+          top: -10px;
+          width: 50%;
+          right: 0;
+        }
+
+        &::after {
+          bottom: -10px;
+          width: 120%;
+          right: 0;
+        }
+      }
+    }
+
+    .navbarLogo {
+      bottom: 20px;
+    }
+
+    .navbarItems {
+      flex-direction: column;
+      width: 100%;
+      height: 40%;
+      order: 2;
+
+      > .col {
+        border-bottom: 0.1px solid #f5deb378;
+        transition: all 1.5s ease-in-out;
+
+        &:nth-of-type(1) {
+          border-top: 0.1px solid #f5deb378;
+        }
+
+        &:hover {
+          border-right: 0.1px solid #f5deb378;
+          border-left: 0.1px solid #f5deb378;
+        }
+
+        a {
+          display: block;
+          width: 100%;
+          height: 100%;
+          padding-top: 10px;
+        }
+      }
+    }
+
+    .iconTrio {
+      order: 1;
+      width: 100%;
+      flex-wrap: nowrap;
+      height: 50px;
+      margin: 0 auto;
+      margin-bottom: 30px;
+
+      .fas,
+      .fab {
+        font-size: 22px;
+      }
+    }
+  }
+
+  .searchBox {
+    width: 460px;
+    margin-left: 45px;
+    position: absolute;
+    transition: all 0.5s linear;
+
+    .searchPreview {
+      padding: 20px;
+
+      .card:nth-of-type(1) {
+        margin-right: 20px;
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 720px) {
+  .searchBox {
+    width: 450px;
+    margin-left: 35px;
+  }
+}
+
+@media only screen and (max-width: 690px) {
+  .searchBox {
+    width: 400px;
+    margin-left: 30px;
+  }
+}
+
+@media only screen and (max-width: 635px) {
+  .searchBox {
+    width: 340px;
+  }
+}
+
+@media only screen and (max-width: 590px) {
+  .searchBox {
+    width: 330px;
+  }
+}
+
+@media only screen and (max-width: 565px) {
+  .searchBox {
+    width: 300px;
+
+    button {
+      font-size: 14px !important;
+    }
+  }
+}
+
+@media only screen and (max-width: 552px) {
+  .searchBox {
+    width: 290px;
+  }
+}
+
+@media only screen and (max-width: 525px) {
+  .searchBox {
+    margin-left: 10px;
+    width: 275px;
+  }
+}
+
+@media only screen and (max-width: 490px) {
+  .searchBox {
+    width: 260px;
+  }
+}
+
+@media only screen and (max-width: 465px) {
+  .searchBox {
+    width: 245px;
+
+    input {
+      font-size: 15px !important;
+    }
+  }
+}
+
+@media only screen and (max-width: 450px) {
+  .searchBox {
+    width: 230px;
+  }
+}
+
+@media only screen and (max-width: 444px) {
+  .searchBox {
```

Comment:
why does line 316 use 450px and line 322 use 444px? this seems redundant.

also, you should avoid magic numbers, if you're using sass  you should define breakpoints
```
$medium: 450px
$large: 800px
```
etc